web/src/containers/Trade/components/_TradeBlock.scss

```diff
@@ -57,6 +57,7 @@ $tabs-title--margin: 2rem;
 
 .trade_block-wrapper {
 	padding: 1rem;
+	max-height: 500px;
```

Comment:
do not use pixel. terrible choice for responsive web. Better you user rem.

Reply:
nice suggestion, The new change is pushed.